service/history/historyReplicator.go

```diff
@@ -348,6 +353,123 @@ func (r *historyReplicator) ApplyEvents(ctx context.Context, request *h.Replicat
 	}
 }
 
+func (r *historyReplicator) checkDuplicatedReplicationEvent(context *workflowExecutionContext, logger bark.Logger) error {
```

Comment:
is this function used anywhere?

Reply:
No, it's already abondoned. I will remove it.